test/test_browser.py

```diff
@@ -3025,7 +3025,7 @@ def test_sdl2_image_formats(self):
     self.btest_exit('test_sdl2_image.c', 600, args=[
       '--preload-file', 'screenshot.jpg',
       '-DSCREENSHOT_DIRNAME="/"', '-DSCREENSHOT_BASENAME="screenshot.jpg"', '-DBITSPERPIXEL=24', '-DNO_PRELOADED',
-      '-sUSE_SDL=2', '-sUSE_SDL_IMAGE=2', '-sSDL2_IMAGE_FORMATS=["jpg"]'
+      '--use-port=sdl2', '--use-port=sdl2_image?formats=["jpg"]'
```

Comment:
I think just `--use-port=sdl2_image?formats=jpg` is cleaner (i.e. drop the quotes and square brackets).

Reply:
100% agree. I just wanted to show that given the regular expression I used, the format of "formats" is actually quite loose and neither the quotes or square brackets are actually required. I didn't know if you preferred it to look like the current settings. The good news is that if somebody copy/paste the syntax of the old settings, it will work as well...